lib/src/next/v1/nu_router.dart

```diff
@@ -204,6 +208,15 @@ abstract class NuRouter implements INuRouter {
   /// Backwards compatible with old routers API
   List<INuRouter> get legacyRouters => [];
 
+  /// Override if you want to wrap the ScreenRoute with another Widget
```

Comment:
Given the `ScreenRoute` class is not exposed by the API, this docstring may look a bit cryptic. Maybe changing to: 
```
... you want to wrap the `builder` function of the NuRoutes registered in this router with another Widget
```